utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -64,12 +64,12 @@ int main(int argc, char **argv)
     int decoderType = atoi(argv[13]);
     int batchSize = atoi(argv[14]);
 
-    bool additionalParamCase = (testCase == 8 || testCase == 21 || testCase == 23 || testCase == 24 || testCase == 49 || testCase == 79);
+    bool additionalParamCase = (testCase == 8 || testCase == 21 || testCase == 23 || testCase == 24 || testCase == 28 || testCase == 49 || testCase == 79);
     bool kernelSizeCase = (testCase == 49);
     bool dualInputCase = (testCase == 2 || testCase == 30 || testCase == 33 || testCase == 61 || testCase == 63 || testCase == 65 || testCase == 68);
     bool randomOutputCase = (testCase == 6 || testCase == 8 || testCase == 84);
-    bool nonQACase = (testCase == 24);
-    bool interpolationTypeCase = (testCase == 21 || testCase == 23 || testCase == 24 || testCase == 79);
+    bool nonQACase = (testCase == 24 || testCase == 28);
```

Comment:
why this is added to nonQA

Reply:
Warp Affine and Perspective alone have some very minor precision related differences that don't match exactly. We's spent considerable cycles trying to match that minor difference, but its something to do with the _mm256_setr_ps instruction on computed float location indexes that differs from rawC. @HazarathKumarM You can put the sertr vs rawC values here.

Reply:
I am confused. Is this going to be fixed?